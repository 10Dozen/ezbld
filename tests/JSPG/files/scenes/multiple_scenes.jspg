# Scene1

This is a first scene.

# Scene2

This is a second scene.

# Scene3

This is a third scene.