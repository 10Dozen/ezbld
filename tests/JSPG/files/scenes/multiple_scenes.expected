[0]
cls = JSPGScene
name = Scene1
desc = This is a first scene.

[1]
cls = JSPGScene
name = Scene2
desc = This is a second scene.

[2]
cls = JSPGScene
name = Scene3
desc = This is a third scene.
